lib/util/search.ts

```diff
@@ -118,12 +118,15 @@ SearchClient.prototype.shouldIndexed = function(page) {
 // BONSAI_URL is following format:
 // => https://{ID}:{PASSWORD}@{HOST}
 SearchClient.prototype.parseUri = function(uri) {
+  if (!uri.startsWith('http')) {
+    throw new Error('URL for Elasticsearch shuold starts with http/https')
```

Comment:
[nits]
`shuold` should be changed to `should`.
```suggestion
    throw new Error('URL for Elasticsearch should starts with http/https')
```